share/doc/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+if(NOT VOTCA_SPHINX_DIR)
+  set(VOTCA_SPHINX_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+endif()
+find_package(Python REQUIRED COMPONENTS Interpreter)
```

Comment:
This is only valid since CMake 3.12 at the moment we support cmake 3.10 @junghans

Reply:
Should we bump it to CMake-3.12? Or do you have machines with CMake-3.10.

Reply:
All the ubuntu 18.04 machines have CMake 3.10

Reply:
@baumeier you have the most machines.

Reply:
@junghans @JensWehner Yes, all of them on 18.04.

Reply:
Workaround added.